src/masternode-budget.cpp

```diff
@@ -1334,6 +1334,25 @@ CBudgetProposal::CBudgetProposal(const CBudgetProposal& other)
     strInvalid = "";
 }
 
+// initialize from network broadcast message
+bool CBudgetProposal::LoadBroadcast(CDataStream& broadcast)
```

Comment:
Maybe better name it `ParseBroadcast` or directly a `SerializeBroadcast`. "Load" means that you are adding it somewhere.